firebase-ml-modeldownloader/src/test/java/com/google/firebase/ml/modeldownloader/internal/CustomModelDownloadServiceTest.java

```diff
@@ -166,7 +169,7 @@ public void downloadService_noHashSuccess() {
                     .withBody(RESPONSE_BODY)));
 
     CustomModelDownloadService service =
-        new CustomModelDownloadService(
+        new CustomModelDownloadService(ApplicationProvider.getApplicationContext(),
```

Comment:
Can you add tests in this file to verify that the headers are set correctly?

Reply:
Will do

Reply:
@vkryachko I have added tests please review when you get a chance.